File: Meghana_data/Backend/trekkingApp/src/main/java/com/example/demo/entities/Payment.java
package com.example.demo.entities;

import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name="payment")
public class Payment {

	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	int paymentid;
	
	@Column
	String trekkername;
	
	@Column
	Date paymentdate=new Date();
	
	@Column
	int noofpersons;
	
	@Column
	double totalcharges;
	
	@Column
	boolean paymentstatus;
}

File: sumit_data/BackEnd_DOTNET/Sahyadri_Navigator_RESTAPI/Models/Payment.cs
using System;
using System.Collections.Generic;

namespace Sahyadri_Navigator_RESTAPI.Models
{
    public partial class Payment
    {
        public int Paymentid { get; set; }
        public int? Noofpersons { get; set; }
        public DateTime? Paymentdate { get; set; }
        public ulong? Paymentstatus { get; set; }
        public double? Totalcharges { get; set; }
        public string? Trekkername { get; set; }
    }
}
